receiver/statsdreceiver/internal/protocol/statsd_parser.go

```diff
@@ -309,6 +319,19 @@ func (p *StatsDParser) Aggregate(line string, addr net.Addr) error {
 			point.SetIntValue(point.IntValue() + parsedMetric.counterValue())
 		}
 
+	case SetType:
+		if existing, ok := instrument.sets[parsedMetric.description]; !ok {
+			instrument.sets[parsedMetric.description] = summaryMetric{
+				points:  []float64{parsedMetric.asFloat}, // TODO: Should this be a different type?
```

Comment:
Why do we put the value as an attribute key? Looking at the definition of set data type, the value should represent a number of unique events between the flushes. So it seems like it can be clearly represented by a gauge.